x-pack/plugins/index_management/public/application/store/selectors/index.js

```diff
@@ -39,14 +40,14 @@ export const getIndexStatusByIndexName = (state, indexName) => {
   const { status } = indices[indexName] || {};
   return status;
 };
-export const getIsSystemIndexByName = (indexNames) => {
+export const getIsSystemIndexByName = (indexNames, allIndices) => {
```

Comment:
Per my [comment](https://github.com/elastic/kibana/pull/112990#issuecomment-926214798) we might want to consider all hidden indices to be system indices.

Reply:
I hear you and at the same time I'd like to understand why we can't have a simple naming convention and stick to it everywhere in the stack. `system` indices are system, and `hidden` indices are simply hidden. Why the UI has to diverge from that and mention to the user that one of his `hidden` index is in fact system index?